lib/metadataTypes/Query.js

```diff
@@ -116,45 +116,129 @@ class Query extends MetadataType {
             this.definition.targetUpdateTypeMapping[metadata.targetUpdateTypeName];
         return metadata;
     }
+    /**
+     * helper for buildDefinitionForExtracts
+     * searches extracted SQL file for template variables and applies the market values
+     *
+     * @param {string} code code from extracted code
+     * @param {TYPE.TemplateMap} templateVariables variables to be replaced in the metadata
+     * @returns {string} code with markets applied
+     */
+    static _applyMarketValues(code, templateVariables) {
+        // fix bad formatting applied by SQL Formatter Plus
+        code = code
```

Comment:
this already exists somewhere else i think, if the same, then please merge into util method